### PR TITLE
Update Rust dependencies

### PR DESCRIPTION
**Pull request by arnaldo2792**
**Issue number:**
N/A

**Description of changes:**
```
tools: update outdated cargo dependencies
cd24de3f sources: update outdated cargo dependencies
d0cea749 tools: update cargo dependencies
97bb2ff4 sources: update cargo dependencies
```

- `nix` had a major version bump, but none of the [Removed](https://github.com/nix-rust/nix/blob/master/CHANGELOG.md) features are being used
- `simplelog` had a major version bump, but no breaking changes were added in [the last release](https://github.com/Drakulix/simplelog.rs/blob/master/CHANGELOG.md)

Things _not_ updated:

- tokio, hyper, hyper-unix-connector, inotify, reqwest, rusoto: refer to #1269 
- gptman : refer to #1196 
- signal-hook: will be handled in a separate PR

**Testing done:**
- aws-k8s-1.18/1.19 aarch64:
    - Launched pods in cluster
    - Checked `systemctl status`, no units failing
    - Checked `journalctl -p3 -xb` no unexpected warnings/failures shown
    - Test APIs behavior by enabling the admin container using user data
    - Unit tests in `sources` and `tools` are passing
- aws-k8s-1.18 x86_64:
   - Same testing
- aws-ecs-1 aarch64/x86_64:
   - Launched a task/service through the ECS console
   - Checked `systemctl status`, no units failing
   - Checked `journalctl -p3 -xb` no unexpected warnings/failures shown
   - Test APIs behavior by enabling the admin container using user data
   - Launched nginx container from within the host, I accessed it through docker and call `curl http://localhost`
- aws-dev aarch64/x86_64:
    - Launched nginx container from within the host, I accessed it through docker and call `curl http://localhost`
   - Checked `systemctl status`, no units failing
   - Checked `journalctl -p3 -xb` no unexpected warnings/failures shown
   - Test APIs behavior by enabling the admin container using user data

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by tjkirch — APPROVED** [review]
Nice!

File: sources/Cargo.lock [hunk 1]
```diff
@@ -3281,9 +3282,9 @@ checksum = "cda74da7e1a664f795bb1f8a87ec406fb89a02522cf6e50620d016add6dbbf5c"
 
 [[package]]
 name = "tokio"
-version = "0.2.24"
+version = "0.2.25"
```
Comment (webern) [inline sources/Cargo.lock:3285]: Only one version of tokio in use. 👍

**Review by webern — APPROVED** [review]
✨

**Review by zmrow — APPROVED** [review]
:seedling:

File: sources/Cargo.lock (hunk 1 above)
Comment (arnaldo2792) [reply sources/Cargo.lock:3285]: :100:

**Pull request merged**